src/attribution/onINP.ts

```diff
@@ -260,6 +262,99 @@ export const onINP = (
     return intersectingLoAFs;
   };
 
+  const getLoAFSummary = (attribution: INPAttribution) => {
+    // Stats across all LoAF entries and scripts.
+    const interactionTime = attribution.interactionTime;
+    const inputDelay = attribution.inputDelay;
+    const processingDuration = attribution.processingDuration;
+    let totalNonForcedStyleAndLayoutDuration = 0;
+    let totalForcedStyleAndLayout = 0;
+    let totalIntersectingScriptsDuration = 0;
+    let numScripts = 0;
+    let slowestScriptDuration = 0;
+    let slowestScriptEntry!: PerformanceScriptTiming;
+    let slowestScriptSubpart!: INPSubpart;
+    const subparts: Partial<
+      Record<INPSubpart, Partial<Record<ScriptInvokerType, number>>>
+    > = {};
+
+    for (const loafEntry of attribution.longAnimationFrameEntries) {
+      totalNonForcedStyleAndLayoutDuration +=
+        loafEntry.startTime +
+        loafEntry.duration -
+        loafEntry.styleAndLayoutStart;
+
+      for (const script of loafEntry.scripts) {
+        const scriptEndTime = script.startTime + script.duration;
+        if (scriptEndTime < interactionTime) {
+          return;
+        }
+        const intersectingScriptDuration =
+          scriptEndTime - Math.max(interactionTime, script.startTime);
+        totalIntersectingScriptsDuration += intersectingScriptDuration;
+        numScripts++;
+        totalForcedStyleAndLayout += script.forcedStyleAndLayoutDuration;
+        const invokerType = script.invokerType;
+        let subpart: INPSubpart = 'processingDuration';
+        if (script.startTime < interactionTime + inputDelay) {
+          subpart = 'inputDelay';
+        } else if (
+          script.startTime >
```

Comment:
```suggestion
          script.startTime >=
```
While testing this, I noticed cases were rAF scripts were showing up in "processingDuration", and I'm pretty sure this is the culprit.

Reply:
BTW, I think it'd be possible to add a test to ensure the expected scripts end up in their expected buckets. I'm testing now just using the `inp.njk` test page and adding a few extra rAF calls as well as a blocking, classic script and I believe it should be possible to do all of that in a test.